_monitoring-your-cluster/pa/reference.md

```diff
@@ -455,225 +531,498 @@ This list is extensive. We recommend using Ctrl/Cmd + F to find what you're look
+   </tr>
+  </tbody>
+ </table>
+
+## Relevant dimensions: `IndexName`, `NodeName`, `ShardType`, `ShardID`
+  
+<table>
+   <thead style="text-align: left">
+   <tr>
+     <th>Metric</th>
+     <th>Description</th>
+   </tr>
+  </thead>
+  <tbody>  
+   <tr>
+     <td>Shard_State 
+     </td>
+     <td>The state of each shard, for example, `STARTED`, `UNASSIGNED`,  and `RELOCATING`.
+     </td>
+   </tr>
+   </tbody>
+</table>
+
+## Relevant dimensions: TBD
+ 
+ <table>
+   <thead style="text-align: left">
     <tr>
-      <td>Shard_State (WIP)
-      </td>
-      <td>IndexName, NodeName, ShardType, ShardID
-      </td>
-      <td>The state of each shard - whether it is STARTED, UNASSIGNED, RELOCATING etc.
-      </td>
+      <th>Metric</th>
+      <th>Description</th>
     </tr>
+   </thead>
+   <tbody>  
+      <tr>
+        <td>LeaderCheck_Latency 
+        </td>
+        <td> TBD
+        </td>
+      </tr>
     <tr>
-      <td>LeaderCheck_Latency (WIP)
+      <td>FollowerCheck_Failure 
       </td>
-      <td rowspan="4">WIP
-      </td>
-      <td rowspan="4">WIP
+      <td> TBD
       </td>
     </tr>
     <tr>
-      <td>FollowerCheck_Failure (WIP)
+      <td>LeaderCheck_Failure 
       </td>
+      <td> TBD
+       </td>
     </tr>
     <tr>
-      <td>LeaderCheck_Failure (WIP)
+      <td>FollowerCheck_Latency
       </td>
-    </tr>
-    <tr>
-      <td>FollowerCheck_Latency (WIP)
+      <td> TBD
       </td>
     </tr>
   </tbody>
 </table>
 
+## Relevant dimensions: `NodeID`, `searchbp_mode`
+The following metrics are new as of OpenSearch 2.11.
+
+<table>
+   <thead style="text-align: left">
+   <tr>
+     <th>Metric</th>
+     <th>Description</th>
+   </tr>
+  </thead>
+  <tbody> 
+     <tr>
+     <td>searchbp_shard_stats_cancellationCount
+     </td>
+     <td>The number of tasks marked for cancellation on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_limitReachedCount
+     </td>
+     <td>The number of times when the number of tasks eligible for cancellation exceeded the set cancellation threshold on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_heap_usage_cancellationCount
+     </td>
+     <td>The number of tasks marked for cancellation because of excessive heap usage since the node last restarted on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_heap_usage_currentMax
+     </td>
+     <td>The maximum heap usage for tasks currently running on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_heap_usage_rollingAvg
+     </td>
+     <td> The rolling average heap usage for n most recent tasks on the shard task. The default value for n is 100.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_cpu_usage_cancellationCount
+     </td>
+     <td>he number of tasks marked for cancellation because of excessive CPU usage since the node last restarted on the shard task.
```

Comment:
```suggestion
     <td>The number of tasks marked for cancellation because of excessive CPU usage since the node last restarted on the shard task.
```